### ISSUE TITLE
Use of Router in bootstrap prevents loading routes in following Plugins

### ISSUE THREAD

**Issue by pyrokar** [CakeDC/users#359]
If you load other Plugins after the users Plugin, Routing does not work.
```
Plugin::load('CakeDC/Users');
Plugin::load('OwnPlugin', ['routes' => true]);
```
This is because the Router is initialized while loading the `config/users.php`. The Router will not reload the routes when loading the second Plugin (see `\Cake\Routing\Router::routes()`)

Maybe it's a Router issue.

A Workaround for this is to reset the initialized flag of the Router:
```
Plugin::load('CakeDC/Users');
\Cake\Routing\Router::$initialized = false;
Plugin::load('OwnPlugin', ['routes' => true]);
```

**Comment by steinkel** [comment]
Interesting, we'll take a deeper look. Thanks for reporting.

**Comment by ajibarra** [comment]
Fixed on #360 . Thanks for reporting.
